### ISSUE TITLE
Fingerbot does not work any more

### ISSUE THREAD

**Issue by flotzke** [ioBroker/ioBroker.zigbee#2287]
### what does not work?

Adapter crashes when trying to use Fingerbot

### Adapter version

1.10.13

### Coordinator firmware version

20230324 ?

### Coordinator

usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20230324193727-if00

### Setup

RaspBerry PI 4

### Debug log

<img width="842" alt="Bildschirmfoto 2024-11-13 um 12 24 38" src="https://github.com/user-attachments/assets/17037063-958a-4414-9e84-0d6077e9be3c">

2024-11-13 12:34:16.778 - info: host.raspberrypi5 stopInstance system.adapter.zigbee.0 (force=false, process=true)
2024-11-13 12:34:16.781 - info: zigbee.0 (23420) Got terminate signal TERMINATE_YOURSELF
2024-11-13 12:34:16.783 - info: zigbee.0 (23420) cleaned everything up...
2024-11-13 12:34:16.784 - info: zigbee.0 (23420) Zigbee: disabling joining new devices.
2024-11-13 12:34:16.886 - info: host.raspberrypi5 stopInstance system.adapter.zigbee.0 send kill signal
2024-11-13 12:34:17.207 - info: zigbee.0 (23420) terminating
2024-11-13 12:34:17.207 - info: zigbee.0 (23420) Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason
2024-11-13 12:34:17.283 - info: zigbee.0 (23420) terminating
2024-11-13 12:34:17.778 - info: host.raspberrypi5 instance system.adapter.zigbee.0 terminated with code 11 (ADAPTER_REQUESTED_TERMINATION)
2024-11-13 12:34:21.057 - info: host.raspberrypi5 instance system.adapter.zigbee.0 in version "1.10.13" started with pid 23442
2024-11-13 12:34:22.726 - info: zigbee.0 (23442) starting. Version 1.10.13 in /opt/iobroker/node_modules/iobroker.zigbee, node: v20.16.0, js-controller: 6.0.11
2024-11-13 12:34:22.763 - info: zigbee.0 (23442) delete old Backup files. keep only last 10
2024-11-13 12:34:22.764 - info: zigbee.0 (23442) Starting Zigbee npm ...
2024-11-13 12:34:22.925 - info: zigbee.0 (23442) Installed Version: iobroker.zigbee@latest
2024-11-13 12:34:27.499 - info: zigbee.0 (23442) Coordinator firmware version: {"type":"EZSP v12","meta":{"product":12,"majorrel":"7","minorrel":"3","maintrel":"1 ","revision":"7.3.1.0 build 176"}}
2024-11-13 12:34:27.499 - info: zigbee.0 (23442) Unable to disable LED, unsupported function.
2024-11-13 12:34:27.499 - info: zigbee.0 (23442) --> transmitPower : high
2024-11-13 12:34:27.508 - info: zigbee.0 (23442) Currently 33 devices are joined:
2024-11-13 12:34:27.548 - info: zigbee.0 (23442) 0xa4c1388faf43e024 (addr 63996): FUT035Z+ - MiBoxer Dual white LED controller (Router)
2024-11-13 12:34:27.550 - info: zigbee.0 (23442) 0x84b4dbfffec39831 (addr 34482): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.551 - info: zigbee.0 (23442) 0xa4c1385bc3bdbeba (addr 10190): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.552 - info: zigbee.0 (23442) 0x00158d0009d0b0f5 (addr 54994): SJCGQ11LM - Aqara Water leak sensor (EndDevice)
2024-11-13 12:34:27.553 - info: zigbee.0 (23442) 0xa4c1384c9643d925 (addr 36852): FUT037Z+ - MiBoxer RGB led controller (Router)
2024-11-13 12:34:27.554 - info: zigbee.0 (23442) 0xa4c13800e14d68ca (addr 27399): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.555 - info: zigbee.0 (23442) 0xa4c138806f0e9921 (addr 48725): FUT037Z+ - MiBoxer RGB led controller (Router)
2024-11-13 12:34:27.556 - info: zigbee.0 (23442) 0x84b4dbfffebfcc61 (addr 55528): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.557 - info: zigbee.0 (23442) 0x540f57fffe0f425b (addr 13624): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.557 - info: zigbee.0 (23442) 0xb43a31fffee1a70f (addr 33778): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:34:27.558 - info: zigbee.0 (23442) 0x0017880103e19763 (addr 7774): 915005106701 - Philips Hue white and color ambiance LightStrip plus (Router)
2024-11-13 12:34:27.560 - info: zigbee.0 (23442) 0xa4c13822d83869e5 (addr 15465): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:34:27.561 - info: zigbee.0 (23442) 0x54ef44100083ebb9 (addr 38761): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:34:27.561 - info: zigbee.0 (23442) 0x54ef44100083ba9c (addr 40012): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:34:27.562 - info: zigbee.0 (23442) -> Configuring 0xa4c13846deaaa883 TS0202
2024-11-13 12:34:27.566 - info: zigbee.0 (23442) 0x54ef441000ac54c1 (addr 18943): RTCGQ14LM - Aqara Motion sensor P1 (EndDevice)
2024-11-13 12:34:27.567 - info: zigbee.0 (23442) 0x0ceff6fffe48eb00 (addr 58876): SNZB-06P - SONOFF Zigbee occupancy sensor (Router)
2024-11-13 12:34:27.567 - info: zigbee.0 (23442) 0x0017880103c027d4 (addr 31663): 8718696449691 - Philips Hue White A60 Single bulb E27/B22 (Router)
2024-11-13 12:34:27.568 - info: zigbee.0 (23442) 0xa4c1383094a0cabf (addr 21592): ZG-204ZL - Tuya Luminance motion sensor (EndDevice)
2024-11-13 12:34:27.569 - info: zigbee.0 (23442) 0xa4c138e688e2be5b (addr 36072): ZG-204ZL - Tuya Luminance motion sensor (EndDevice)
2024-11-13 12:34:27.569 - info: zigbee.0 (23442) 0xa4c1388095c0a770 (addr 41123): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:34:27.570 - info: zigbee.0 (23442) 0xa4c13846deaaa883 (addr 63697): 809WZT - Tuya Motion sensor (EndDevice)
2024-11-13 12:34:27.570 - info: zigbee.0 (23442) 0xa81710b984cd0000 (addr 15010): CK-BL702-AL-01 - Tuya Zigbee LED bulb (Router)
2024-11-13 12:34:27.571 - info: zigbee.0 (23442) 0xf082c0fffec031db (addr 45187): FUT089Z - MiBoxer RGB+CCT Remote (EndDevice)
2024-11-13 12:34:27.571 - info: zigbee.0 (23442) 0xa4c138f1867a2e28 (addr 59090): RSH-HS06_1 - Tuya Temperature & humidity sensor (EndDevice)
2024-11-13 12:34:27.572 - info: zigbee.0 (23442) 0xa4c13885193ed66f (addr 14545): RSH-HS06_1 - Tuya Temperature & humidity sensor (EndDevice)
2024-11-13 12:34:27.573 - info: zigbee.0 (23442) 0x70b3d52b600fcd1d (addr 17222): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:34:27.573 - info: zigbee.0 (23442) 0xa4c138b13d68179e (addr 35344): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:34:27.574 - info: zigbee.0 (23442) 0x70b3d52b6011481a (addr 15972): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:34:27.575 - info: zigbee.0 (23442) 0x70b3d52b600fce55 (addr 22232): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:34:27.577 - info: zigbee.0 (23442) 0xa4c1383eb9a81c8d (addr 40608): ZY-M100-24GV3 - Tuya 24G MmWave radar human presence motion sensor（added distance switch） (Router)
2024-11-13 12:34:27.577 - info: zigbee.0 (23442) 0xa4c138cd4e108a13 (addr 44933): ZY-M100-24GV3 - Tuya 24G MmWave radar human presence motion sensor（added distance switch） (Router)
2024-11-13 12:34:27.578 - info: zigbee.0 (23442) 0x54ef441000843c20 (addr 41742): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:34:27.578 - info: zigbee.0 (23442) 0xa4c13802f06a6bf3 (addr 56186): TS0001_fingerbot - Tuya Zigbee fingerbot plus (EndDevice)
2024-11-13 12:34:27.579 - info: zigbee.0 (23442) Zigbee started
2024-11-13 12:34:27.581 - warn: zigbee.0 (23442) download icon from https://www.zigbee2mqtt.io/images/devices/FUT035Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT035Z+.png
2024-11-13 12:34:27.639 - warn: zigbee.0 (23442) download icon from https://www.zigbee2mqtt.io/images/devices/FUT037Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:34:27.646 - warn: zigbee.0 (23442) download icon from https://www.zigbee2mqtt.io/images/devices/FUT037Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:34:27.719 - warn: zigbee.0 (23442) DeviceConfigure: 0xa4c13846deaaa883 TS0202 Failed to configure. When device works is all fine when not wake up the device and check again
2024-11-13 12:34:27.733 - info: zigbee.0 (23442) debug devices set to []
2024-11-13 12:34:27.842 - warn: zigbee.0 (23442) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:34:27.845 - warn: zigbee.0 (23442) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:34:27.847 - warn: zigbee.0 (23442) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT035Z+.png
2024-11-13 12:34:48.779 - error: zigbee.0 (23442) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2024-11-13 12:34:48.780 - error: zigbee.0 (23442) unhandled promise rejection: Cannot read properties of undefined (reading 'includes')
2024-11-13 12:34:48.780 - error: zigbee.0 (23442) TypeError: Cannot read properties of undefined (reading 'includes')
at /opt/iobroker/node_modules/iobroker.zigbee/main.js:726:31
at Array.forEach ()
at Zigbee.publishFromState (/opt/iobroker/node_modules/iobroker.zigbee/main.js:652:23)
2024-11-13 12:34:48.780 - error: zigbee.0 (23442) Cannot read properties of undefined (reading 'includes')
2024-11-13 12:34:48.782 - info: zigbee.0 (23442) cleaned everything up...
2024-11-13 12:34:48.784 - info: zigbee.0 (23442) Zigbee: disabling joining new devices.
2024-11-13 12:34:49.282 - info: zigbee.0 (23442) terminating
2024-11-13 12:34:49.283 - warn: zigbee.0 (23442) Terminated (UNCAUGHT_EXCEPTION): Without reason
2024-11-13 12:34:49.844 - error: host.raspberrypi5 Caught by controller[1]: zhc:lixee: Was not able to detect the Linky linky_mode. Default to historique
2024-11-13 12:34:49.844 - error: host.raspberrypi5 Caught by controller[1]: zhc:lixee: Was not able to detect the Linky energy_phase. Default to single_phase
2024-11-13 12:34:49.844 - error: host.raspberrypi5 Caught by controller[1]: zhc:lixee: Not able to detect the current tarif. Not filtering any expose...
2024-11-13 12:34:49.844 - error: host.raspberrypi5 Caught by controller[2]: TICMeter: Exposes: No endpoint
2024-11-13 12:34:49.844 - error: host.raspberrypi5 Caught by controller[3]: zh:ezsp: 'ezsp' driver is deprecated and will only remain to provide support for older firmware (pre 7.4.x). Migration to 'ember' is recommended. If using Zigbee2MQTT see https://github.com/Koenkk/zigbee2mqtt/discussions/21462
2024-11-13 12:34:49.844 - error: host.raspberrypi5 instance system.adapter.zigbee.0 terminated with code 6 (UNCAUGHT_EXCEPTION)
2024-11-13 12:34:49.844 - info: host.raspberrypi5 Restart adapter system.adapter.zigbee.0 because enabled
2024-11-13 12:35:00.004 - info: javascript.0 (1180) script.js.common.global: DTU heartbeat (min): 5.25355
2024-11-13 12:35:00.004 - info: javascript.0 (1180) script.js.common.global: Keine Daten vom INVERTER seit: 315213
2024-11-13 12:35:00.004 - info: javascript.0 (1180) script.js.common.global: REBOOT DTU...
2024-11-13 12:35:20.214 - info: host.raspberrypi5 instance system.adapter.zigbee.0 in version "1.10.13" started with pid 23458
2024-11-13 12:35:21.910 - info: zigbee.0 (23458) starting. Version 1.10.13 in /opt/iobroker/node_modules/iobroker.zigbee, node: v20.16.0, js-controller: 6.0.11
2024-11-13 12:35:21.948 - info: zigbee.0 (23458) delete old Backup files. keep only last 10
2024-11-13 12:35:21.949 - info: zigbee.0 (23458) Starting Zigbee npm ...
2024-11-13 12:35:22.090 - info: zigbee.0 (23458) Installed Version: iobroker.zigbee@latest
2024-11-13 12:35:26.680 - info: zigbee.0 (23458) Coordinator firmware version: {"type":"EZSP v12","meta":{"product":12,"majorrel":"7","minorrel":"3","maintrel":"1 ","revision":"7.3.1.0 build 176"}}
2024-11-13 12:35:26.681 - info: zigbee.0 (23458) Unable to disable LED, unsupported function.
2024-11-13 12:35:26.681 - info: zigbee.0 (23458) --> transmitPower : high
2024-11-13 12:35:26.692 - info: zigbee.0 (23458) Currently 33 devices are joined:
2024-11-13 12:35:26.742 - info: zigbee.0 (23458) 0xa4c1388faf43e024 (addr 63996): FUT035Z+ - MiBoxer Dual white LED controller (Router)
2024-11-13 12:35:26.745 - info: zigbee.0 (23458) 0x84b4dbfffec39831 (addr 34482): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.747 - info: zigbee.0 (23458) 0xa4c1385bc3bdbeba (addr 10190): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.747 - info: zigbee.0 (23458) 0x00158d0009d0b0f5 (addr 54994): SJCGQ11LM - Aqara Water leak sensor (EndDevice)
2024-11-13 12:35:26.749 - info: zigbee.0 (23458) 0xa4c1384c9643d925 (addr 36852): FUT037Z+ - MiBoxer RGB led controller (Router)
2024-11-13 12:35:26.750 - info: zigbee.0 (23458) 0xa4c13800e14d68ca (addr 27399): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.751 - info: zigbee.0 (23458) 0xa4c138806f0e9921 (addr 48725): FUT037Z+ - MiBoxer RGB led controller (Router)
2024-11-13 12:35:26.753 - info: zigbee.0 (23458) 0x84b4dbfffebfcc61 (addr 55528): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.754 - info: zigbee.0 (23458) 0x540f57fffe0f425b (addr 13624): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.755 - info: zigbee.0 (23458) 0xb43a31fffee1a70f (addr 33778): TS0505B_1 - Tuya Zigbee RGB+CCT light (Router)
2024-11-13 12:35:26.756 - info: zigbee.0 (23458) 0x0017880103e19763 (addr 7774): 915005106701 - Philips Hue white and color ambiance LightStrip plus (Router)
2024-11-13 12:35:26.757 - info: zigbee.0 (23458) 0xa4c13822d83869e5 (addr 15465): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:35:26.759 - info: zigbee.0 (23458) 0x54ef44100083ebb9 (addr 38761): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:35:26.760 - info: zigbee.0 (23458) 0x54ef44100083ba9c (addr 40012): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:35:26.760 - info: zigbee.0 (23458) -> Configuring 0xa4c13846deaaa883 TS0202
2024-11-13 12:35:26.765 - info: zigbee.0 (23458) 0x54ef441000ac54c1 (addr 18943): RTCGQ14LM - Aqara Motion sensor P1 (EndDevice)
2024-11-13 12:35:26.766 - info: zigbee.0 (23458) 0x0ceff6fffe48eb00 (addr 58876): SNZB-06P - SONOFF Zigbee occupancy sensor (Router)
2024-11-13 12:35:26.767 - info: zigbee.0 (23458) 0x0017880103c027d4 (addr 31663): 8718696449691 - Philips Hue White A60 Single bulb E27/B22 (Router)
2024-11-13 12:35:26.768 - info: zigbee.0 (23458) 0xa4c1383094a0cabf (addr 21592): ZG-204ZL - Tuya Luminance motion sensor (EndDevice)
2024-11-13 12:35:26.769 - info: zigbee.0 (23458) 0xa4c138e688e2be5b (addr 36072): ZG-204ZL - Tuya Luminance motion sensor (EndDevice)
2024-11-13 12:35:26.770 - info: zigbee.0 (23458) 0xa4c1388095c0a770 (addr 41123): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:35:26.770 - info: zigbee.0 (23458) 0xa4c13846deaaa883 (addr 63697): 809WZT - Tuya Motion sensor (EndDevice)
2024-11-13 12:35:26.771 - info: zigbee.0 (23458) 0xa81710b984cd0000 (addr 15010): CK-BL702-AL-01 - Tuya Zigbee LED bulb (Router)
2024-11-13 12:35:26.772 - info: zigbee.0 (23458) 0xf082c0fffec031db (addr 45187): FUT089Z - MiBoxer RGB+CCT Remote (EndDevice)
2024-11-13 12:35:26.772 - info: zigbee.0 (23458) 0xa4c138f1867a2e28 (addr 59090): RSH-HS06_1 - Tuya Temperature & humidity sensor (EndDevice)
2024-11-13 12:35:26.773 - info: zigbee.0 (23458) 0xa4c13885193ed66f (addr 14545): RSH-HS06_1 - Tuya Temperature & humidity sensor (EndDevice)
2024-11-13 12:35:26.775 - info: zigbee.0 (23458) 0x70b3d52b600fcd1d (addr 17222): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:35:26.775 - info: zigbee.0 (23458) 0xa4c138b13d68179e (addr 35344): WHD02 - Tuya Wall switch module (Router)
2024-11-13 12:35:26.776 - info: zigbee.0 (23458) 0x70b3d52b6011481a (addr 15972): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:35:26.777 - info: zigbee.0 (23458) 0x70b3d52b600fce55 (addr 22232): TS011F_plug_3 - Tuya Smart plug (with power monitoring by polling) (Router)
2024-11-13 12:35:26.780 - info: zigbee.0 (23458) 0xa4c1383eb9a81c8d (addr 40608): ZY-M100-24GV3 - Tuya 24G MmWave radar human presence motion sensor（added distance switch） (Router)
2024-11-13 12:35:26.781 - info: zigbee.0 (23458) 0xa4c138cd4e108a13 (addr 44933): ZY-M100-24GV3 - Tuya 24G MmWave radar human presence motion sensor（added distance switch） (Router)
2024-11-13 12:35:26.781 - info: zigbee.0 (23458) 0x54ef441000843c20 (addr 41742): MCCGQ14LM - Aqara Door and window sensor E1 (EndDevice)
2024-11-13 12:35:26.782 - info: zigbee.0 (23458) 0xa4c13802f06a6bf3 (addr 56186): TS0001_fingerbot - Tuya Zigbee fingerbot plus (EndDevice)
2024-11-13 12:35:26.782 - info: zigbee.0 (23458) Zigbee started
2024-11-13 12:35:26.785 - warn: zigbee.0 (23458) download icon from https://www.zigbee2mqtt.io/images/devices/FUT035Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT035Z+.png
2024-11-13 12:35:26.846 - warn: zigbee.0 (23458) download icon from https://www.zigbee2mqtt.io/images/devices/FUT037Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:35:26.854 - warn: zigbee.0 (23458) download icon from https://www.zigbee2mqtt.io/images/devices/FUT037Z-.png saved into /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:35:26.912 - info: zigbee.0 (23458) debug devices set to []
2024-11-13 12:35:26.916 - warn: zigbee.0 (23458) DeviceConfigure: 0xa4c13846deaaa883 TS0202 Failed to configure. When device works is all fine when not wake up the device and check again
2024-11-13 12:35:27.047 - warn: zigbee.0 (23458) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT035Z+.png
2024-11-13 12:35:27.051 - warn: zigbee.0 (23458) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:35:27.340 - warn: zigbee.0 (23458) ERROR : icon path not found /opt/iobroker/node_modules/iobroker.zigbee/admin/img/FUT037Z+.png
2024-11-13 12:35:29.989 - info: javascript.0 (1180) script.js.common.global: Bürotemp: 19.1
2024-11-13 12:35:58.263 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0x84b4dbfffec39831 TS0505B
2024-11-13 12:35:58.429 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xa4c1385bc3bdbeba TS0505B
2024-11-13 12:35:58.656 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xa4c1384c9643d925 TS0505B
2024-11-13 12:35:58.826 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xa4c13800e14d68ca TS0505B
2024-11-13 12:35:58.873 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xa4c138806f0e9921 TS0505B
2024-11-13 12:35:58.968 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0x84b4dbfffebfcc61 TS0505B
2024-11-13 12:35:59.166 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0x540f57fffe0f425b TS0505B
2024-11-13 12:36:07.820 - info: javascript.0 (1180) script.js.common.global: Draussen: 7.5
2024-11-13 12:36:07.820 - info: javascript.0 (1180) script.js.common.global: Drinnen: 18.7
2024-11-13 12:36:07.820 - info: javascript.0 (1180) script.js.common.global: BalkontürOpen: false
2024-11-13 12:36:07.867 - info: javascript.0 (1180) script.js.common.global: Wind: 2.4
2024-11-13 12:36:08.271 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xb43a31fffee1a70f TS0505B
2024-11-13 12:36:17.568 - warn: zigbee.0 (23458) DeviceAvailability:Failed to ping 0xa4c13822d83869e5 TS0001
2024-11-13 12:37:30.819 - info: javascript.0 (1180) script.js.common.global: Bürotemp: 19.2
2024-11-13 12:37:58.773 - info: opendtu.0 (1561) Connect to OpenDTU over websocket connection.
2024-11-13 12:38:07.825 - info: javascript.0 (1180) script.js.common.global: Draussen: 7.7
2024-11-13 12:38:07.825 - info: javascript.0 (1180) script.js.common.global: Drinnen: 18.7
2024-11-13 12:38:07.826 - info: javascript.0 (1180) script.js.common.global: BalkontürOpen: false
2024-11-13 12:38:07.873 - info: javascript.0 (1180) script.js.common.global: Wind: 2.4

NodeJs 20.16.0
NPM 10.8.1

**Comment by asgothian** [comment]
Please confirm: the crash occurs once after install - after restart the adapter runs stable with no crashes ?

that’s how the log looks.

also: please place the log entry in code tags (edit the post, select the log entries and use the <> button to activate code tags.

**Comment by j-r79** [comment]
Same here. Everytime i trigger fingerbot the adapter crashes. Not only after install, each time fingerbot is triggered.

`
zigbee.0
2024-11-14 08:08:23.471	error	Cannot read properties of undefined (reading 'includes')

zigbee.0
2024-11-14 08:08:23.471	error	TypeError: Cannot read properties of undefined (reading 'includes') at /opt/iobroker/node_modules/iobroker.zigbee/main.js:726:31 at Array.forEach (<anonymous>) at Zigbee.publishFromState (/opt/iobroker/node_modules/iobroker.zigbee/main.js:652:23)

zigbee.0
2024-11-14 08:08:23.470	error	unhandled promise rejection: Cannot read properties of undefined (reading 'includes')

zigbee.0
2024-11-14 08:08:23.470	error	Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().

zigbee.0
2024-11-14 08:08:23.467	debug	Type of toZigbee is 'object', Contains key false

zigbee.0
2024-11-14 08:08:23.467	debug	Setting' converter to converter with key(s)'["state","on_time","off_wait_time"]}

zigbee.0
2024-11-14 08:08:23.467	debug	Type of toZigbee is 'object', Contains key ["state","on_time","off_wait_time"]

zigbee.0
2024-11-14 08:08:23.464	debug	entity: 0xa4c1381001fa1e5e TS0001_fingerbot_1 {"type":"device","device":{"_events":{},"_eventsCount":0,"_maxListeners":100,"ID":2,"_applicationVersion":82,"_dateCode":"","_endpoints":[{"_events":{},"_eventsCount":0,"_maxListeners":100,"deviceID":0,"inputClusters":[6,61184,0],"outputClusters":[25,10],"profileID":260,"ID":1,"clusters":{"genOnOff":{"attributes":{"onOff":1}},"genBasic":{"attributes":{"65503":"7��-\r","65506":54,"65508":0,"65534":0,"modelId":"TS0001","manufacturerName":"_TZ3210_dse8ogfy","powerSource":3,"zclVersion":3,"appVersion":82,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"deviceIeeeAddress":"0xa4c1381001fa1e5e","deviceNetworkAddress":54986,"_binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0026b86b9d","endpointID":1}],"_configuredReportings":[],"meta":{},"pendingRequests":{"sendInProgress":false,"ID":1,"deviceIeeeAddress":"0xa4c1381001fa1e5e"}}],"_hardwareVersion":1,"_ieeeAddr":"0xa4c1381001fa1e5e","_interviewCompleted":true,"_interviewing":false,"_lastSeen":1731559216371,"_manufacturerID":4417,"_manufacturerName":"_TZ3210_dse8ogfy","_modelID":"TS0001","_networkAddress":54986,"_powerSource":"Battery","_stackVersion":0,"_type":"EndDevice","_zclVersion":3,"_skipDefaultResponse":false,"_pendingRequestTimeout":0,"_customClusters":{},"meta":{"configured":1003603311}},"mapped":{"fingerprint":[{"modelID":"TS0001","manufacturerName":"_TZ3210_dse8ogfy"},{"modelID":"TS0001","manufacturerName":"_TZ3210_j4pdtz9v"}],"model":"TS0001_fingerbot_1","vendor":"Adaprox","description":"Zigbee fingerbot plus","whiteLabel":[{"vendor":"Adaprox","model":"TS0001_fingerbot_1","description":"Zigbee fingerbot plus","fingerprint":[{"manufacturerName":"_TZ3210_dse8ogfy"}]}],"fromZigbee":[{"cluster":"genOnOff","type":["attributeReport","readResponse"],"options":[{"name":"state_action","label":"State action","access":2,"type":"binary","property":"state_action","description":"State actions will also be published as 'action' when true (default false).","value_on":true,"value_off":false}]},{"cluster":"manuSpecificTuya","type":["commandDataResponse","commandDataReport","commandActiveStatusReport","commandActiveStatusReportAlt"]}],"toZigbee":[{"key":["state","on_time","off_wait_time"]},{},{"key":["scene_store"]},{"key":["scene_recall"]},{"key":["scene_add"]},{"key":["scene_remove"]},{"key":["scene_remove_all"]},{"key":["scene_rename"]},{"key":["read"]},{"key":["write"]},{"key":["command"]},{"key":["reset"]},{"key":["zclcommand"]}],"exposes":[{"type":"switch","features":[{"name":"state","label":"State","access":7,"type":"binary","property":"state","description":"On/off state of the switch","value_on":"ON","value_off":"OFF","value_toggle":"TOGGLE"}]},{"name":"battery","label":"Battery","access":1,"type":"numeric","property":"battery","description":"Remaining battery in %, can take up to 24 hours before reported","category":"diagnostic","unit":"%","value_max":100,"value_min":0},{"name":"mode","label":"Mode","access":3,"type":"enum","property":"mode","description":"Working mode","values":["click","switch","program"]},{"name":"lower","label":"Lower","access":3,"type":"numeric","property":"lower","description":"Down movement limit","unit":"%","value_max":100,"value_min":50,"value_step":1},{"name":"upper","label":"Upper","access":3,"type":"numeric","property":"upper","description":"Up movement limit","unit":"%","value_max":50,"value_min":0,"value_step":1},{"name":"delay","label":"Delay","access":3,"type":"numeric","property":"delay","description":"Sustain time","unit":"s","value_max":10,"value_min":0,"value_step":1},{"name":"reverse","label":"Reverse","access":3,"type":"binary","property":"reverse","description":"Reverse","value_on":"ON","value_off":"OFF"},{"name":"touch","label":"Touch","access":3,"type":"binary","property":"touch","description":"Touch controll","value_on":"ON","value_off":"OFF"},{"name":"linkquality","label":"Linkquality","access":1,"type":"numeric","property":"linkquality","description":"Link quality (signal strength)","category":"diagnostic","unit":"lqi","value_max":255,"value_min":0}],"meta":{"tuyaSendCommand":"sendData","tuyaDatapoints":[[101,"mode",{}],[102,"lower",{}],[103,"delay","[Circular]"],[104,"reverse",{}],[105,"battery","[Circular]"],[106,"upper","[Circular]"],[107,"touch",{}],[109,"program","[Circular]"]]},"options":["[Circular]"]},"endpoint":"[Circular]","endpoints":"[Circular]","name":"0xa4c1381001fa1e5e"}

zigbee.0
2024-11-14 08:08:23.463	debug	publishFromState : 0xa4c1381001fa1e5e TS0001_fingerbot_1 [{"stateDesc":{"id":"state","prop":"state","name":"On/off state of the switch","role":"switch","write":true,"read":true,"type":"boolean","setattr":"state","readable":true,"writable":true},"value":true,"index":0,"timeout":0}]`

**Comment by asgothian** [comment]
I have identified a possible problem, but it will take me a few days to verify and post the fix. With a bit of luck, It will be available on the weekend.

A.

**Comment by asgothian** [comment]
I did edit a fix, but am unable to test it before the weekend. You are welcome to install from [my GitHub repo](https://github.com/asgothian/ioBroker.zigbee) in order to test said fix.

Again - this fix is provided as is and is not tested in any way. Use at your own risk. If the adapter does not run, just install from the default GitHub repository. Tests will be done automatically over the weekend.

A.

**Comment by asgothian** [comment]
General tests of the changes are complete. Please verify if the linked version does fix the fingerbot issue. I cannot test this due to lack of access to a device.

**Comment by flotzke** [comment]
error | Starting zigbee-herdsman problem : "Failure send version:{\"type\":\"Buffer\",\"data\":[0,0,0,4]}"
-- | --

**Comment by asgothian** [comment]
Please post more than a single error line when posting errors. This one line tells me nothing.
Also: Please install the current 1.1013 GitHub version and verify this is running before trying again to install my fork.


A.

**Comment by Haui74** [comment]
@asgothian habe deine Version istalliert, bis jetzt ohne Fehler und die Bots funktionieren wieder

**Comment by asgothian** [comment]
Fix merged into the official 1.10.13 version.

Please use the official github version when using the fingerbot.
